DfE.FindInformationAcademiesTrusts/Pages/Trusts/_SourceList.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@model ITrustsAreaModel
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      <span>Source and updates</span>
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    @foreach (var source in Model.DataSources)
+    {
+      <dl role="list">
+        <dt class="govuk-body" role="listitem">
+          <span class="govuk-!-font-weight-bold margin-right-class">@string.Join(", ", source.Fields): </span> @Model.MapDataSourceToName(source.DataSource)
+        </dt>
+        <dd class="govuk-body" role="definition">Last updated: @(source.DataSource.LastUpdated is null ? "Unknown" : source.DataSource.LastUpdated.Value.ToString(ViewConstants.ViewDateFormat))</dd>
```

Comment:
I think the definition role is definitely not what we want here. 

> The definition ARIA role can be included on an element that is a definition of a term or concept, similar to the native <dfn> element

However we're using the dl as a list of things with descriptions